tests/CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ ConfigureTest(STATIC_MULTIMAP_TEST
     static_multimap/custom_pair_retrieve_test.cu
     static_multimap/custom_type_test.cu
     static_multimap/heterogeneous_lookup_test.cu
-    static_multimap/insert_test.cu
+    static_multimap/insert_contains_test.cu
```

Comment:
I was a bit confused at first when reading the test name but I guess it makes sense in order to reduce code duplication.

Reply:
I cannot find an easy way to test `insert` without using `contains` and vice versa. :smiling_face_with_tear: